source/tool/chipsec/helper/linux/helper.py

```diff
@@ -109,15 +136,17 @@ def start(self, start_driver):
             logger().log("[helper] Linux Helper started/loaded")
 
     def stop( self ):
+        self.close()
+        subprocess.call(["rmmod", self.MODULE_NAME])
```

Comment:
Could we use `check_call` instead and log an error if the unloading failed? (We might need to add a condition to test if /dev/chipsec is there before running that. Similarly to what is done in `create`)